src/s1reader/s1_burst_slc.py

```diff
@@ -911,15 +911,21 @@ def az_fm_rate_mismatch_from_llh(self,
                              for datetime_vec in self.extended_coeffs.dc_aztime_vec]
 
         # calculate splined interpolation of the coeffs. and tau_0s
+        if len(fm_rate_aztime_sec_vec) <=1:
```

Comment:
```suggestion
        if len(fm_rate_aztime_sec_vec) <= 1:
```
nit - formatting. Same below